sources/packages/backend/apps/api/src/types/config.ts

```diff
@@ -127,3 +128,7 @@ export interface ESDCIntegrationConfig {
 export interface SFASIntegrationConfig {
   ftpReceiveFolder: string;
 }
+
+export interface InstitutionIntegrationConfig {
```

Comment:
can belong to the library.

Reply:
I am moving all the configs to the library as part of the PR https://github.com/bcgov/SIMS/pull/1538. I can move this one once the PR is merged also.